### DISCUSSION TITLE
Rest API not giving the desired data according to time filter applied for Pull requests

### DISCUSSION THREAD

**Discussion by AutomationUIUser** [community/community#57412] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

When trying to access the Pull Requests from the API, updated in the specific time frame, not getting the desired output.

https://api.github.com/search/issues?page=1&per_page=100&q=is:pr+repo:AutomationUIUser/ProfilingTestRepository2+updated:2019-12-26T23:30:00Z..2020-01-27T22:49:48Z&sort=updated&order=asc

There are 2100 closed pull requests available at this repository which are updated in this time frame. But this API is giving the 1827 Pull requests as a response.

Also, API is not giving the requested Ascending order.

I have also applied this filter on the GitHub UI and it is also giving the same result.

**Comment by bselwe** [comment]
This is an issue in [Test-buzz](https://github.com/Test-buzz/test-repo/pulls) repositories as well - it seems to affect all of our organizations. Basically, the API fails to return some PRs based on `updated_at`, e.g. when using `updated:>=2023-06-01` the API does not return some PRs updated in 2023.06.
